sdv/errors.py

```diff
@@ -56,5 +56,9 @@ def __str__(self):
         )
 
 
+class InvalidDataTypeError(Exception):
+    """Error to raise if data type is not valid."""
```

Comment:
The `InvalidDataError` only accepts lists of errors and automatically formats it as: "The metadata has the following errors: ..." son instead I added a new error type.